src/Components/Facility/FacilityHome.tsx

```diff
@@ -80,76 +73,99 @@ export const FacilityHome = (props: any) => {
 
   useMessageListener((data) => console.log(data));
 
-  const fetchData = useCallback(
-    async (status: statusType) => {
+  const { res: permittedFacilityRes, data: permittedFacilityData } = useQuery(
+    routes.getPermittedFacility,
+    {
+      pathParams: {
+        id: facilityId,
+      },
+    }
+  );
+
+  const {
+    res: capacityRes,
+    data: capacityFetchData,
+    refetch: capacityFetch,
+  } = useQuery(routes.getCapacity, {
+    pathParams: { facilityId },
+  });
+
+  const {
+    res: doctorRes,
+    data: doctorFetchData,
+    refetch: doctorFetch,
+  } = useQuery(routes.listDoctor, {
+    pathParams: { facilityId: facilityId },
+  });
```

Comment:
I'd say, consider breaking down the doctor capacity / bed capacity / triage to seperate components and move the data fetching down to those components. That way you won't have to deal with having multiple useQuery in same component too and would be a much better cleaner state management.